src/main/java/org/broadinstitute/hellbender/tools/walkers/variantutils/UpdateVCFSequenceDictionary.java

```diff
@@ -24,6 +24,36 @@
 /**
  * Updates the sequence dictionary in a variant file using the dictionary from a variant, alignment, reference,
  * or dictionary file.
+ *
+ * <p>This tool is designed to update the sequence dictionary in a variant file using a dictionary from another variant,
+ * alignment, dictionary, or reference file. The dictionary must be valid, i.e. must contain a sequence record, for all
+ * variants in the target file.
+ * </p>
+ *
+ * <h3>Example</h3>
+ *
+ * <pre>
+ * gatk UpdateVCFSequenceDictionary \
+ *   -O updated_sequence_dictionary.dict \
+ *   -V variants.vcf \
+ *   --source-dictionary input_dictionary.dict \
+ *   --replace=true
+ * </pre>
+ *
+ * <pre>
+ * gatk UpdateVCFSequenceDictionary \
+ *   -O updated_sequence_dictionary.dict \
+ *   -R reference.fa \
+ *   -V variants.vcf \
+ *   --replace=true
+ * </pre>
+ *
+ * <p>
+ *    The -O argument specifies the name of the updated directory file. The --source-dictionary argument specifies the
+ *    input sequence dictionary. The --replace argument is optional, and forces the replacement of the dictionary
+ *    if the input file already has a dictionary.
+ * </p>
+ *
```

Comment:
Again, program group should be VCFManipulationProgramGroup.java.

Reply:
I get the same error in IntelliJ...